### PR TITLE
Fix the block range chunking for syncing

### PR DESCRIPTION
**Pull request by pgebheim**

### CODE REVIEW

File: packages/augur-sdk/src/state/sync/BulkSyncStrategy.ts [hunk 1]
```diff
@@ -20,30 +21,24 @@ export class BulkSyncStrategy extends AbstractSyncStrategy implements SyncStrate
     endBlockNumber: number,
   ): Promise<number> {
     if(!endBlockNumber) throw new Error('Cannot bulk sync forever. Please pass in an endblock');
+    if(startBlockNumber >= endBlockNumber) throw new Error('Starting point of bulk sync must be less than the endpoint')
```
Comment (justinbarry) [inline packages/augur-sdk/src/state/sync/BulkSyncStrategy.ts]: Is there a convoluted set of circumstances where these could be *equal*?

File: packages/augur-utils/src/generators.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export function* chunkRange(start, end, chunk) {
+   for (;start < end;start+=chunk) {
```
Comment (justinbarry) [inline packages/augur-utils/src/generators.ts]:
```suggestion
   for (;start =< end;start+=chunk) {
```

image
The top has this patch, the second is as written here.

**Review by justinbarry — CHANGES_REQUESTED** [review]
One dumb question and one feedback item.

File: packages/augur-sdk/src/state/sync/BulkSyncStrategy.ts (hunk 1 above)
Comment (pgebheim) [reply packages/augur-sdk/src/state/sync/BulkSyncStrategy.ts]: Perhaps? I suppose with your change below that would be fine

Comment (pgebheim) [inline packages/augur-sdk/src/state/sync/BulkSyncStrategy.ts]:
```suggestion
    if(startBlockNumber > endBlockNumber) throw new Error('Starting point of bulk sync must be less than or equal to the endpoint')
```

**Review by justinbarry — APPROVED** [review]
LGTM

**Pull request merged**